File: tools/src/main/java/com/theapp/tools/Logger.java
package com.theapp.tools;

import android.support.annotation.NonNull;

public interface Logger {

    void d(@NonNull String tag, @NonNull String msg);

    void e(@NonNull String tag, @NonNull String msg);

    void e(@NonNull String tag, @NonNull String msg, @NonNull Throwable tr);

    void i(@NonNull String tag, @NonNull String msg);

    void v(@NonNull String tag, @NonNull String msg);

    void w(@NonNull String tag, @NonNull String msg);

}

File: tools/src/main/java/com/theapp/tools/Logger.kt
package com.theapp.tools

interface Logger {

    fun d(tag: String, msg: String)

    fun e(tag: String, msg: String)

    fun e(tag: String, msg: String, tr: Throwable)

    fun i(tag: String, msg: String)

    fun v(tag: String, msg: String)

    fun w(tag: String, msg: String)

}
